src/ProcessManager/ChromeManager.php

```diff
@@ -108,13 +108,35 @@ private function getDefaultArguments(): array
         return $args;
     }
 
+    private function createProcess(string $chromeDriverBinary): Process
```

Comment:
Maybe could we simplify this and be more flexible by just creating a `chromedriver_arguments` key allowing to pass an array of random arguments to ChromeDriver?

Reply:
I agree, that would be a better solution. I will update this PR later today.